docs/source/array-design.rst

```diff
@@ -66,31 +66,32 @@ key-value pairs required to eventually compute the desired values, for example
    }
 
 The name of an ``Array`` object can be found in the ``name`` attribute.  One
-can get a nested list of keys with the ``.__dask_keys__()`` method.  One can
-flatten down this list with ``dask.array.core.flatten()``; this is sometimes
+can get a nested list of keys with the ``.__dask_keys__()`` method.  Additionally, 
+one can flatten down this list with ``dask.array.core.flatten()``. This is sometimes
 useful when building new dictionaries.
 
 Chunks
 ------
 
-We also store the size of each block along each axis.  This is a tuple of
-tuples such that the length of the outer tuple is equal to the dimension and
-the lengths of the inner tuples are equal to the number of blocks along each
-dimension.  In the example illustrated above this value is as follows::
+We also store the size of each block along each axis.  This is composed of 
+a tuple of tuples such that the length of the outer tuple is equal to the 
+number of dimensions of the array, and the lengths of the inner tuples are 
+equal to the number of blocks along each dimension.  In the example illustrated 
```

Comment:
Overall this paragraph is improved. Thanks for that.

Would suggest we change `blocks` here to `chunks` for clarity.

Reply:
Throughout the text it is used mostly blocks instead of chunks, but since section refers to chunks, it makes sense to replace `blocks` for `chunks` for clarity. What do you think?

Reply:
Good point. Missed that. Happy to defer this to you.

Reply:
To avoid inducing any confusion, I would keep the use of `blocks` instead of `chunks` in this section.